packages/social-trader/src/components/horizontal-list-shadow-container/horizontal-shadow-list.tsx

```diff
@@ -11,7 +11,8 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
 
 const ListContainer = styled.div<{ withScroll?: boolean }>`
   display: flex;
-  padding-bottom: ${$paddingXsmall}px;
+  padding-bottom: ${({ withScroll = true }) =>
+    withScroll ? `${$paddingXsmall}px` : ""};
```

Comment:
HorizontalShadowList is used not only in the dashboard. This change will affect other places as well.